### PR TITLE
Authentication & Authorization

### PR DESCRIPTION
**Pull request by stefanorosanelli**
Authentication and authorization paragraph with brief endpoint permissions explanation

### CODE REVIEW

File: source/glossary.rst [hunk 1]
```diff
@@ -2,18 +2,55 @@ Glossary
 ===========
 
 .. glossary::
+    :sorted:
+
+    access token
+        A string granted by the authorization server used to identify the issuer of a request.
+        The access token has to be sent to the resource server every time that the client want
```
Comment (didoda) [inline source/glossary.rst]: minor correction: "the client wants [...]" or "the clients want [...]"

File: source/glossary.rst [hunk 2]
```diff
@@ -2,18 +2,55 @@ Glossary
...
+    JWT
+        JSON Web Tokens are an open, industry standard `RFC 7519 <https://tools.ietf.org/html/rfc7519>`_
+        method for representing claims securely between two parties.
+
+        A JWT is composed by three parts:
+
+        * an **header** containing informations about the token type and algorithm used. It is Base64URL encoded.
+        * a **payload** containing informations in the form of claims (informations we want to transmit).
+          It is Base64URL encoded.
+        * a **signature** used to verify the authenticity of the JWT using an valid algorithm defined by
```
Comment (didoda) [inline source/glossary.rst]: "a valid algorithm"

File: source/glossary.rst [hunk 3]
```diff
@@ -2,18 +2,55 @@ Glossary
...
+          `JSON Web Signature (JWS) <https://tools.ietf.org/html/rfc7515>`_
+          specification (for example a shared secret `HMAC <https://tools.ietf.org/html/rfc7515#page-36>`_).
+
+        More info `here <http://jwt.io>`_.
+
 
     object
-        an object in BEdita is the atomic content of your project's data, it could be of different type like a document, an event, an images,
+        An object in BEdita is the atomic content of your project's data, it could be of different type like a document, an event, an images,
```
Comment (didoda) [inline source/glossary.rst]: "an image,"

File: source/glossary.rst [hunk 4]
```diff
@@ -2,18 +2,55 @@ Glossary
...
-        a role in RBAC permission model (https://en.wikipedia.org/wiki/Role-based_access_control)
+        A role in RBAC permission model (https://en.wikipedia.org/wiki/Role-based_access_control)
         is used to assign permissions to perform some operations
 
+
     user
-        project users with credentials to login; usually they consist of a username and passowrd or some external
-        auth provider identifiers and profile data like first name, surname, contact information etc;
-        each user has at least one role, used to grant permissions on some operations;
+        Actual project users with credentials to login; main attributes are username and passowrd or some external
```
Comment (didoda) [inline source/glossary.rst]: "password"

**Review by didoda — CHANGES_REQUESTED** [review]
Some minor fixes on text

File: source/glossary.rst (hunk 1 above)
Comment (stefanorosanelli) [reply source/glossary.rst]: done

File: source/glossary.rst (hunk 2 above)
Comment (stefanorosanelli) [reply source/glossary.rst]: done

File: source/glossary.rst (hunk 3 above)
Comment (stefanorosanelli) [reply source/glossary.rst]: done

File: source/glossary.rst (hunk 4 above)
Comment (stefanorosanelli) [reply source/glossary.rst]: done

**Review by didoda — APPROVED** [review]
ok

**Pull request merged**